pyproject.toml

```diff
@@ -0,0 +1,37 @@
+# Choosing a build backend:
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "transit_equity"
+version = "0.0.0"
+authors = [
+    {name = "Himanshu Naidu", email = "hnaidu36@uw.edu"},
+    {name = "Rebecca Hayes", email = "rhayes7@uw.edu"},
+    {name = "Siman Ning", email = "tatasky@uw.edu"},
+    {name = "Ishan Saksena", email = "saksena7@uw.edu"},
+    {name = "Bryna Hazelton", email = "brynah@uw.edu"},
+    {name = "Ryan Avery", email = "rpavery@uw.edu"}
+]
+description = "Python Package for Transit Equity on ORCA Fare Card Data"
+readme = "README.md"
+dependencies = [
+    "numpy>=2.0.0",
+    "pandas>=2.2.2",
+    "matplotlib>=3.8.4",
+    "pgcli>=4.1.0",
+    "psycopg2>=2.9.9",
```

Comment:
It seems that to install this using pip (`pip install -e  .`), the Windows Linux Subsystem needs to install gcc (for psycopg2).
But when installing psycopg2 through conda/mamba, we don't get an issue without gcc, because conda/mamba install the following:
`The following NEW packages will be INSTALLED:

  keyutils           conda-forge/linux-64::keyutils-1.6.1-h166bdaf_0 
  krb5               conda-forge/linux-64::krb5-1.21.3-h659f571_0 
  libedit            conda-forge/linux-64::libedit-3.1.20191231-he28a2e2_2 
  libpq              conda-forge/linux-64::libpq-16.3-ha72fbe1_0 
  libstdcxx-ng       conda-forge/linux-64::libstdcxx-ng-14.1.0-hc0a3c3a_0 
  psycopg2           conda-forge/linux-64::psycopg2-2.9.9-py312h08590aa_0 
  python_abi         conda-forge/linux-64::python_abi-3.12-4_cp312 `

Reply:
If you are managing your environment using conda/mamba you should run `pip install --no-deps -e .` so that pip doesn't mess up the dependencies. It's best not to let pip mess with the conda/mamba installation.

Reply:
Got it. That makes sense. 
I assume we will have to test on how this works in a pure Python environment (or virtualenv) as well, for comprehensiveness. 
Just to make sure the setup works everywhere. 

Currently, this has been tested by me only for the following 2 approaches, both related to conda/mamba:
1. (Assuming gcc is installed) Create a conda environment, and then directly install transit_equity with dependencies with `pip install -e .`
2. Create a conda environment using orca.yaml and then install transit_equity with `pip install --no-deps -e .`

Reply:
This is fine for now. Later if we want we can set up some continuous integration jobs to test all of these things.

Reply:
For everyone else's reference, please follow the Method 2. in your own environments, except instead of orca.yaml, use the environment.yaml in the root directory of this repository.
We will have to remember to keep the environment.yaml file updated.